templates/crane.conf.erb

```diff
@@ -5,5 +5,5 @@
 [general]
 debug: false
 #data_dir_polling_interval: 60
-data_dir: /var/www/pub/docker/app
+data_dir: /var/lib/pulp/published/docker/v2/app/
```

Comment:
katello will only have v2 support, but i was thinking more as a default for the crane module.  @parthaa why not make it parametrized and just set the default to /var/lib/pulp/published/docker/v2/app/

That way users of the module can change it but it will default to what works for docker v2 which seems more ideal as docker v1 support will be dropped from pulp before too long.